sdk/core/core-amqp/package.json

```diff
@@ -69,7 +69,7 @@
     "util": "^0.11.1"
   },
   "peerDependencies": {
-    "rhea-promise": "^0.1.15"
+    "rhea-promise": "^1.0.0"
   },
   "devDependencies": {
     "@azure/identity": "^1.0.0-preview.1",
```

Comment:
Please rebase your branch onto current master.  This dev dependency should be `1.0.0-preview.2`.